decidim-admin/app/views/decidim/admin/participatory_process_steps/edit.html.erb

```diff
@@ -0,0 +1,11 @@
+<% content_for :title do %>
+  <h2><%= t ".title" %></h2>
+<% end %>
+
+<%= form_for(@form, url: participatory_process_step_path(@participatory_process_step.participatory_process, @participatory_process_step)) do |f| %>
```

Comment:
I tried it and I couldn't get to generate the correct URL. Will try again, just in case I forgot something.